src/Compilers/Helpers/GlobalAssemblyCacheHelpers/GacFileResolver.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.CodeAnalysis.Scripting
 {
     /// <summary>
-    /// Extends <see cref="MetadataFileReferenceResolver"/> to enable resolution of assembly
+    /// Extends MetadataFileReferenceResolver to enable resolution of assembly
```

Comment:
@jasonmalinowski I talked at length with the compiler lead and he just ignored me.